lib/puppet/type/apache_directive.rb

```diff
@@ -31,6 +31,11 @@ def insync?(is)
     isnamevar
   end
 
+  newparam(:directive) do
+    desc 'The apache directive to modify'
+    defaultto self[:name]
```

Comment:
The Travis CI builds are failing because of this line. What you need instead is:

```ruby
defaultto { @resource[:name] }
```